### PR TITLE
Angular health module

### PR DESCRIPTION
**Pull request by DamnClin**
Related to https://github.com/jhipster/jhipster-lite/issues/2240

### CODE REVIEW

File: src/main/resources/generator/client/angular/admin/src/main/webapp/app/admin/health/health.component.html.mustache [hunk 1]
```diff
@@ -10,12 +10,12 @@
 
     <ng-container matColumnDef="key">
       <th mat-header-cell *matHeaderCellDef>Service name</th>
-      <td mat-cell *matCellDef="let element">{{=<% %>=}}{{ element.key }}<%={{ }}=%></td>
+      <td mat-cell *matCellDef="let element">{{ element.key }}</td>
```
Comment (qmonmert) [inline src/main/resources/generator/client/angular/admin/src/main/webapp/app/admin/health/health.component.html.mustache:13]: are you sure about that? mustaches are ok in the HTML?

Comment (DamnClin) [reply src/main/resources/generator/client/angular/admin/src/main/webapp/app/admin/health/health.component.html.mustache:13]: Yep, in modules we only replace the mustache keys we know, unknown keys are ignored and left without any updates

Comment (DamnClin) [reply src/main/resources/generator/client/angular/admin/src/main/webapp/app/admin/health/health.component.html.mustache:13]: (I generated the app with the old and new module to see that I had to update the templates :))

Comment (qmonmert) [reply src/main/resources/generator/client/angular/admin/src/main/webapp/app/admin/health/health.component.html.mustache:13]: ok thanks 🙂

**Pull request merged**